### ISSUE TITLE
NvimTreeFindFile no longer working

### ISSUE THREAD

**Issue by skbolton** [kyazdani42/nvim-tree.lua#638]
I have tried to update my config based on the new `setup` function. Since updating the `NvimTreeFindFile` command no longer works. Doesn't throw any erros but doesn't open the sidebar up. `NvimTreeToggle` seems to be working fine.

Here is my config for nvim-tree
```
local tree = require('nvim-tree')

vim.g.nvim_tree_show_icons = {
   git = 0,
   folders = 1,
   files = 1,
};

vim.g.nvim_tree_icons = {
   default = '',
   symlink = '',
   git = {
     unstaged = "✗",
     staged = "✓",
     unmerged = "",
     renamed = "➜",
     untracked = "★"
   },
   folder = {
     default = " ",
     open = " "
   }
}

vim.g.nvim_tree_width_allow_resize = true;

tree.setup {
  disable_netrw = false
}
```

**Comment by CantoroMC** [comment]
Same here, i guess we should be a little patience for a couple of days, till the author finishes to refactor and port the configuration to lua.

**Comment by kyazdani42** [comment]
should be fixed in `1.3.2` (0ca87178b62c81ef7f)
@CantoroMC indent_markers are properly rendered on my end, would you mind opening a new issue with more informations ?

**Comment by towry** [comment]
Still not work, run `:NvimTreeFindFile` nothing happen.
